offset_request.go

```diff
@@ -1,19 +1,7 @@
 package sarama
 
-// OffsetTime is used in Offset Requests to ask for all messages before a certain time. Any positive int64
-// value will be interpreted as milliseconds, or use the special constants defined here.
-type OffsetTime int64
-
-const (
-	// LatestOffsets askes for the latest offsets.
-	LatestOffsets OffsetTime = -1
```

Comment:
it means different things for the `offsetRequestBlock` because it's used for multiple offsets, so it's a starting point, not a single offset (thus the old comment "Note that because offsets are pulled in descending order, asking for the earliest offset will always return you a single element." because the other choices would return multiple).